neo/rawio/blackrockrawio.py

```diff
@@ -108,7 +108,8 @@ class BlackrockRawIO(BaseRawIO):
             ID of nsX file from which to load data, e.g., if set to
             5 only data from the ns5 file are loaded. If None, the Io
             will take the maximum if file is present.
-            Contrary to previsous version of the IO:
+            Contrary to previous version of the IO:
+
```

Comment:
I am discovering that this doc is outdated.
Could you change this here it would avoid another PR specific to that:

```
        nsx_to_load (int, list, 'max', 'all' (=None)) default None:
            IDs of nsX file from which to load data, e.g., if set to
            5 only data from the ns5 file are loaded. 
            If 'all', then all nsX will be loaded.
            Contrary to previsous version of the IO  (<0.7), nsx_to_load
            must be set at the init before parse_header().
```

Reply:
Does the commit I just pushed match what you want?

Reply:
Yes perfect thank you.